src/main/java/com/exadel/discountwebapp/vendor/mapper/VendorMapper.java

```diff
@@ -16,6 +16,7 @@
 public class VendorMapper {
     private final LocationService locationService;
     private final ModelMapper modelMapper = new ModelMapper();
+    private final LocationMapper locationMapper = new LocationMapper();
```

Comment:
LocationMapper is a Spring component, so it should be constructed only by Spring and injected here

Reply:
done